### PR TITLE
[kibana] Run as 1000

### PR DESCRIPTION
**Pull request by naseemkullah**
- [x] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [x] README.md updated with any new values or changes
- [ ] Updated template tests in `${CHART}/tests/*.py` 
- [ ] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

### CODE REVIEW

File: kibana/templates/deployment.yaml [hunk 1]
```diff
@@ -63,6 +63,9 @@ spec:
     {{- end }}
       containers:
       - name: kibana
+        securityContext:
```
Comment (Crazybus) [inline kibana/templates/deployment.yaml]: Same comment as the Elasticsearch PR. It makes the most sense to expose the securityContext to be configurable just like with the pod level securityContext.

https://github.com/elastic/helm-charts/blob/26fe7570fb668b909d11cddb6c5fa2f8cfaed288/kibana/templates/deployment.yaml#L30-L33

Comment (naseemkullah) [reply kibana/templates/deployment.yaml]: Sounds good! I've added it as a configurable var. I'll go ahead and do it the same in ES to keep it consistent.

**Comment by naseemkullah** [comment]
I took the liberty of adding extra `securityContext` as it is a best practice. I was unable to use a RO filesystem however, even when mounting an `emptyDir` volume on `/usr/share/kibana/optimize`.

File: kibana/values.yaml [hunk 2]
```diff
@@ -44,8 +44,17 @@ kibanaConfig: {}
 #       nestedkey: value
 
 # If Pod Security Policy in use it may be required to specify security context as well as service account
-podSecurityContext: {}
-  #runAsUser: "place the user id here"
+
+podSecurityContext:
+  fsGroup: 2000
```
Comment (Crazybus) [inline kibana/values.yaml]: What is group id 2000? It doesn't exist in the kibana docker image. The kibana group has id 1000.

**Comment by Crazybus** [comment]
jenkins test this please

File: kibana/values.yaml (hunk 2 above)
Comment (naseemkullah) [reply kibana/values.yaml]: Thought I'd use a number that does not map to an actual group (still works) but yes let's go with 1000 that maps to an existing group instead!

**Comment by Crazybus** [comment]
jenkins test this please

**Comment by Crazybus** [comment]
Thank you so much for adding this in! 🔐

**Comment by naseemkullah** [comment]
My pleasure!

**Pull request merged**